metatron-app/metatron-discovery
# Fn fix exploredata error

### Description

3023
Fix show hiding tag
2971
fix navigate to detail view error
**Related Issue** : 

METATRON-3023
METATRON-2971

### How Has This Been Tested?

2971
1. Explore data -> click data
2. check if navigation works well

3023
1. Explore data 
2. check tags are showing properly
#### Need additional checks?


### Types of changes



### Checklist:


### Additional Context
